smartcar_frontend/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
What does this do? Is it something that is necessary in general?

Reply:
I honestly don't know. I assume it was added by the npx create-react-app command or something similar. Google will be consulted.